attributes/default.rb

```diff
@@ -42,18 +42,27 @@
 default['datadog']['agent_version'] = nil # nil to install latest
 # Agent flavor to install, acceptable values are "datadog-agent", "datadog-iot-agent"
 default['datadog']['agent_flavor'] = 'datadog-agent' # "datadog-agent" to install the datadog-agent package
+# default['datadog']['fips_proxy_version'] = '0.1.0'
```

Comment:
Is there a reason why this line is commented out? I think we should do the same thing as we do for agent_version by default, which would be setting it to `nil`.